tests/angular_devkit/build_angular/hello-world-app/src/main.server.ts

```diff
@@ -5,6 +5,7 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
+import '@angular/localize';
```

Comment:
Is this needed in this PR?

Reply:
Since the FW version is updated to 9.0.0-next.5+ with the yarn upgrade, it is. (The versions in the package file were `~9.0.0-next.4`.  Note the tilde.) These changes are in a separate commit along with setting the minimum to `9.0.0-next.5` to avoid confusion and is also the first version with `@angular/localize`.

Reply:
Removed the next.6 update from this PR.